cloud_functions/README.md

```diff
@@ -11,5 +11,5 @@ This is a collection of cloud functions that augment the core bard service. The
 
 ## Deployment
 ```sh
-gcloud --project terra-bard-[env] functions deploy flagCryptominer --runtime nodejs12 --trigger-topic cryptominers
+gcloud --project terra-bard-[env] functions deploy flagCryptominer --runtime nodejs12 --trigger-http
```

Comment:
TOL: Now that this is http, I assume the authentication is configured in the console, and enforced outside this code, right?

Reply:
Yes. Authenticated is the default when using `gcloud functions deploy`; there would need to be an additional command-line option to make it open to the world. Access is controlled via permissions in cloud console.